.github/workflows/move-issues-to-in-progress.yaml

```diff
@@ -37,11 +32,9 @@ jobs:
           popd &>/dev/null
 
       - name: Checkout code to allow hub to communicate with the project
-        if: ${{ !contains(github.event.pull_request.labels.*.name, 'force-skip-ci') }}
         uses: actions/checkout@v2
 
       - name: Move issue to "In progress"
-        if: ${{ !contains(github.event.pull_request.labels.*.name, 'force-skip-ci') }}
         env:
           GITHUB_TOKEN: ${{ secrets.KATA_GITHUB_ACTIONS_TOKEN }}
```

Comment:
This will stop working.

Reply:
Did this ever work then? According to the documentation no runner gets secrets when triggered from a forked repo, it doesn't say that it applies only to workflow calls AFAICT.

Reply:
I don't think this ever worked.